tests/unit/common/test-util.js

```diff
@@ -8,6 +8,8 @@
 		checkClass: function (element, cls) {
 			if (QUnit && typeof (QUnit.ok) === 'function') {
 				QUnit.ok(element.hasClass(cls), 'The control with id: ' + element[0].id + ' does not contain the class: ' + cls);
+			} else if (QUnit && QUnit.assert && typeof(QUnit.assert.ok) === 'function') {
+				QUnit.assert.ok(element.hasClass(cls), 'The control with id: ' + element[0].id + ' does not contain the class: ' + cls);
```

Comment:
I think this will just have to get `assert` as parameter, rather than trying the global functions.
Would this even properly count towards the per-test count?

Reply:
Indeed, checked and the global `assert` has no test context but it falls back to a stored current so the result gets pushed to the correct test. Could've sworn it failed before, oh well 🤷‍♂️ 
Will take this as it is, though the old version can probably be removed later on.
// CC @wnvko for similar changes